docker-compose.production.yaml

```diff
@@ -0,0 +1,17 @@
+services:
+  badgehub-frontend:
+    image: ghcr.io/badgeteam/badgehub-frontend:${FRONTEND_IMAGE_TAG:-latest}
+    ports:
+      - "8001:3000"
+    restart: always
```

Comment:
With this restart policy, is it still possible to stop the container without deleting it?